src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
```

Comment:
```suggestion
public class Lottery {
    private Random random = new Random();
    
    public static Ball getRandomBall() {
        int value = random.nextInt(100);
```
якщо нам не потрібен новий об'єкт класу Random кожного разу то краще винести цю змінну на рівень класу